README.md

```diff
@@ -1,28 +1,69 @@
 # cached-module-loader
 
-Highly experimental bundler and loader of Node.js modules.
+> Caching bundler and loader of Node.js modules using `vm` module's `cachedData` feature implemented since Node.js v6.
```

Comment:
No need for the quote.

To rephrase a little:

> Caching bundler and loader of Node.js modules which uses the `cachedData` feature in the `vm` module (available since Node.js v6).